api/services/mailer/templates/layout.js

```diff
@@ -1,16 +1,106 @@
-function layout(content) {
+const { app: { hostname } } = require('../../../../config');
+
+const IMAGE_HOST_DOMAIN = `${hostname}/images/transactional-emails`;
+
+const FOOTER_TEXT = 'cloud.gov is a product of the General Services Administration, 1800 F Street NW, Washington, DC 20405';
+
+// common styles
+const cssForPs = () => 'font-size:16px;line-height:26px;margin:16px 0;';
```

Comment:
It took me a bit to realize what the `Ps` and `As` meant. For better semantics what if we export an object with the common `p` and `a` css. In the future, it should be straight forward to add other element css styles.

ie
```js
// declare
const css = {
  a: 'font-size:16px;line-height:26px;margin:16px 0;',
  p: 'color:#067df7;text-decoration:none;',
  h1: ...
}

// import and use in other templates
<a style="{layout.css.a}"></a>
```

Reply:
I like this suggestion.